index.html

```diff
@@ -1,8 +1,11 @@
 <!DOCTYPE html>
 <!--
+			  /\_/\
+			 ( o.o )
+		      > ^ <
 	  The main page for Hacklahoma.
       TODO: Fix fonts on IE
- -->
```

Comment:
We should specify pixel stuff too. It's so the link preview looks good on FB & Twitter and whatnot